README.md

```diff
@@ -1,6 +1,11 @@
-# `teal`: Interactive Exploratory Data Analysis with Shiny Web-Applications <a href='https://insightsengineering.github.io/teal/'><img src="man/figures/teal.png" align="right" height="139" style="max-width: 100%; max-height: 139px;"/></a  >
+# `teal`: Interactive exploratory data analysis with `shiny` web-applications <a href='https://insightsengineering.github.io/teal/'><img src="man/figures/teal.png" align="right" height="139" style="max-width: 100%; max-height: 139px;"/></a  >
```

Comment:
Titles in english are [capitalized](https://www.grammarly.com/blog/capitalization-in-the-titles/)

Reply:
done [a6b7638](https://github.com/insightsengineering/teal/pull/1021/commits/a6b763840a36673983e3608dcd72c1fb6d6a241c)